WunderWeather/weather.py

```diff
@@ -140,6 +140,23 @@ def __init__(self, api_key, settings=None):
 
         self.format = 'json'
 
+    def alerts(self, query):
+        """Shorthand to interface with the conditions data feature
```

Comment:
This should change to be alerts data feature not conditions, maybe copied from today_now feature?